src/main/java/com/mulberry/blackmulberry/controller/main/OrdersController.java

```diff
@@ -6,15 +6,17 @@
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.ModelMap;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
 
 import java.util.List;
 
 @Controller
 @RequiredArgsConstructor
+@RequestMapping("orders")
 public class OrdersController {
     private final ProductRepository productRepository;
 
-    @GetMapping("/orders")
+    @GetMapping("/")
     public String ordersPage(ModelMap modelMap){
         // todo find all products in orders
         List<Product> products = productRepository.findAll();
```

Comment:
please dont write logic in the controller and dont use repository here. create productService and invoke logic execution from there